Lang/French.pj.Lang

```diff
@@ -517,3 +517,45 @@
 #2055#  "Les graphismes LLE ne sont pas à utiliser par tout le monde !!!\nIl est recommandé d’utiliser seulement ceci à des fins de test et non pour jouer.\n\nChanger pour les graphismes en mode LLE ?"
 #2056#  "Émulation audio haut niveau (HLE)"
 #2057#  "L’audio HLE nécessite un plugin tiers !!!\nSi vous n’utilisez pas un plugin audio tiers qui supporte le HLE, vous n’entendrez aucun son.\n\nChanger pour l’audio en mode HLE ?"
+
+
+/*********************************************************************************
+* Android                                                                        *
+*********************************************************************************/
+#3000#  "Paramètres"
+#3001#  "Aide/Forum"
+#3002#  "Rapporter un Problème"
+#3003#  "A Propos De"
+#3004#  "Jeu Récemment Joué"
+#3005#  "Jeux"
+#3006#  "Répertoire du Jeu"
+#3007#  "Choisir un dossier à scanner"
+#3008#  "Inclure les sous répertoires"
+#3009#  "Dossier Parent"
+#3010#  "Répertoires"
+#3011#  "Mémoire Interne"
+#3012#  "Scan..."
+#3013#  "OK"
+#3014#  "Annuler"
+#3015#  "Information"
+#3016#  "Project64 pour Android"
+#3017#  "Licence"
+#3018#  "Révision"
+#3019#  "Project64 pour Android\u2122 est un port de la version Windows de project64. La version\Android u2122 peut lancer la plupart des jeux N64."

```

Comment:
@BenjaminSiskoo : Second "\u2122" is incorrect, probably an unexpected drag'n drop :)